packages/signers/src/turnkey-secp256k1-signer.ts

```diff
@@ -0,0 +1,124 @@
+import { TurnkeyClient } from "@turnkey/http";
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+import { ethers } from "ethers"; // v6 helpers
+import 'dotenv/config';
+import type { Signer } from "@sovereign-sdk/signers";
+import { hexToBytes, bytesToHex } from "@sovereign-sdk/utils";
+import { Point } from "@noble/secp256k1";
+import { keccak_256 } from "@noble/hashes/sha3";
+
+// --- Turnkey Configuration ---
+const ORG_ID = process.env.TURNKEY_ORG_ID;
```

Comment:
Could we please move all the env vars to a configuration object instead. This way we don't control how and where these need to be configured we just receive them as parameters

Same as what the actual turnkey client does: https://github.com/Sovereign-Labs/sovereign-sdk-web3-js/pull/189/files#diff-79afd446c73470ec4aea9d8ad8aa8ba905213d5323795579e1c7ad5f78530665R49